python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb

```diff
@@ -16,27 +16,21 @@
    "metadata": {},
    "source": [
     "## Setup\n",
-    "To install the nuScenes-lidarseg expansion, please download the dataset from https://www.nuscenes.org/download. Upon unzipping, you should see two items:\n",
-    "- `lidarseg_v1.0-mini` \n",
-    "- `v1.0-mini`\n",
-    "\n",
-    "Copy both the `lidarseg_v1.0-mini` folder (this contains `.bin` files; each .bin file contains the label of each point in the point cloud) and `v1.0-mini` into `/data/sets/nuscenes`. If you already have an existing `v1.0-mini` folder from nuScenes, it is safe to replace it as the `v1.0-mini` from nuScenes-lidarseg will have the required files as well. \n",
-    "\n",
-    "If you have downloaded `v1.0-trainval` and `v1.0-test` for nuScenes-lidarseg as well, place (or replace) them in `/data/sets/nuscenes`. \n",
-    "\n",
-    "Essentially, your `/data/sets/nuscenes` folder should end up looking like this:\n",
+    "To install the nuScenes-lidarseg expansion, please download the dataset from https://www.nuscenes.org/download. Unpack the compressed file(s) into `/data/sets/nuscenes` and your folder structure should end up looking like this:\n",
     "```\n",
     "└── nuscenes  \n",
     "    ├── Usual nuscenes folders (i.e. samples, sweep)\n",
     "    │\n",
-    "    ├── lidarseg_v1.0-{mini, trainval} <- Contains the .bin files (Note that v1.0-test does not \n",
-    "    │                                     have any .bin files associated with it.\n",
-    "    │\n",
+    "    ├── lidarseg_v1.0-{mini, trainval} <- Contains the .bin files; each .bin file contains the label \n",
+    "    │                                      of each point in the point cloud (note that v1.0-test does \n",
+    "    │                                      not have any .bin files associated with it)                          \n",
```

Comment:
Try to break the lines earlier, so the jupyter notebook in Github does not break them for you:
https://github.com/nutonomy/nuscenes-devkit/blob/4da7c01512ad2741474cd96bcfe53e0d54847639/python-sdk/tutorials/nuscenes_lidarseg_tutorial.ipynb